jest.config.integration.js

```diff
@@ -0,0 +1,7 @@
+const defaultConfig = require('./jest.config');
+
+module.exports = {
+  ...defaultConfig,
+  testMatch: ['**/test/integration/**/*.test.ts'],
+  testTimeout: 10000,
```

Comment:
Is there a reason this is less than the default?

Reply:
No, let's use default then